src/pages/popup/Popup.tsx

```diff
@@ -73,7 +72,7 @@ const router = createMemoryRouter(routes);
 
 export const Popup = () => {
   return (
-    <LazyMotion features={domAnimation} strict>
+    <>
```

Comment:
Avoiding nesting changes here. We can replace this fragment with `React.StrictMode` in future.